pages/linux/scrot.md

```diff
@@ -0,0 +1,15 @@
+# scrot
+
+> Screen capture utility.
+
+- Capture a screenshot:
+
+`scrot`
+
+- Capture a screenshot interactively:
+
+`scrot --select`
+
+- Capture a screenshot from the currently focused window:
+
+`scrot --focused`
```

Comment:
How about adding an example with a filename.
```suggestion
`scrot --focused {{capture.png}}`
```

Reply:
Perhaps we could do that in a separate example, @mfrw?